lib/SILOptimizer/Mandatory/TFPartition.cpp

```diff
@@ -2743,6 +2743,28 @@ SILFunction *PartitionCloner::lookupSendReceiveFunction(StringRef fnName,
   return fn;
 }
 
+/// A wrapper over SwitchEnumInst, which treats its optional default case just
+/// like the other cases.
+class SwitchEnumCaseIterator {
+public:
+    SwitchEnumCaseIterator(SwitchEnumInst *inst) : inst(inst) {}
```

Comment:
Wrong indentation

Reply:
I think this can use the methods on TermInst like getSuccessorBlocks()

Reply:
Done.